tools/changelog.py

```diff
@@ -40,9 +40,8 @@
 from git import Repo
 from github import Github
 
-if sys.version_info.major < 3:
-    UTF8Writer = codecs.getwriter('utf8')
-    sys.stdout = UTF8Writer(sys.stdout)
+if sys.version_info[:2] >= (3, 6):
+    raise RuntimeError("Python version must be >= 3.6")
```

Comment:
Maybe we can leave this one in for a while, it has saved me more than a few times when I try to run tests with the wrong virtualenv

Reply:
The error, or the thing that was here before?

Reply:
ahh, sorry, never mind

Reply:
I think the logic here is reversed

Reply:
ping

Reply:
Good catch, fixed now.